lisa/notifiers/junit.py

```diff
@@ -302,18 +310,27 @@ def _add_test_case_result(
 
         if message.status == TestStatus.FAILED:
             failure = ET.SubElement(testcase, "failure")
-            failure.attrib["message"] = message.message
-            failure.text = message.stacktrace
-
+            if runbook.include_failed_messages:
+                failure.attrib["message"] = message.message
+                # failure.text = message.stacktrace
+                if str(message.stacktrace) != "":
+                    failure.attrib["stacktrace"] = str(message.stacktrace)
             testsuite_info.failed_count += 1
```

Comment:
Why does it need force convert to string?

Reply:
For some weird reason, linting is complaining about typecasting. I forced a string typecast just to be sure.